README.md

```diff
@@ -51,19 +52,28 @@
 
 * [Bidali](https://giftcards.bidali.com/buy-giftcards-with-dai/) - Buy gift cards with Dai.
 
-## Finance, Investment, and Trading
+## Finance, Investment
 
 * [Compound](https://compound.finance/) - Earn interest on Dai via algorithmic Money Markets.
 * [Dharma](https://www.dharma.io) - Lend your Dai and earn interest.
 * [Maker](https://cdp.makerdao.com/) - Get loans in Dai against collateralized ETH.
-* [Uniswap](https://uniswap.io/) - Decentralized protocol for swapping ETH, Dai and other tokens.
+* [Nuo](https://app.nuo.network/) - Simplest way to margin trade.
+
+## Trading
 
+* [Uniswap](https://uniswap.io/) - Decentralized protocol for swapping ETH, Dai and other tokens.
+* [KyberSwap](https://kyberswap.com/) - A simple way to exchange tokens. No deposit, oo orderbook, competitive spreads.
+* [token.store ETH](https://token.store/) - Decentralized exchange for ERC-20 tokens with an open smart contract. Features on-chain order-matching and settlement.
+* [dYdX](https://trade.dydx.exchange/) - The most powerful open trading platform for crypto assets.
+* [Neutral](https://dapp.neutralproject.com/) - Neutral Dollar is a metastable basket of stablecoins. A digital dollar with lower volatility and globally shared liquidity.
 
 ## Freelancers
 
 ## Other
 
 * [Gitcoin](https://gitcoin.co/) - Platform for crowdfunding and freelance developers for your software projects.
+* [Living Room of Satoshi](https://www.livingroomofsatoshi.com/) - Pay with confidence at Australia's largest crypto payments gateway.
+* [CelerX](https://celerx.app/) - Play eSports games, win crypto prizes. Have fun. Play to earn!
```

Comment:
Nice additions! 👍  Again, only request would be to use alphabetical order.

Reply:
Fixed in https://github.com/apbendi/use-dai/pull/2/commits/55bf547833a3795f45d711ec79fd8d62fdddbbeb